cdap-client/src/main/java/co/cask/cdap/client/AuthorizationClient.java

```diff
@@ -203,8 +207,12 @@ private HttpResponse executeRequest(HttpRequest request, int... allowedErrorCode
     HttpResponse response = restClient.execute(request, config.getAccessToken(), allowedErrorCodes);
     if (HttpURLConnection.HTTP_FORBIDDEN == response.getResponseCode()) {
       // TODO:(CDAP-5302) Include the logged in username here
-      throw new UnauthorizedException(String.format("Unauthorized to perform %s %s with body %s",
-                                                    request.getMethod(), request.getURL(), request.getBody()));
+      InputSupplier<? extends InputStream> requestBody = request.getBody();
+      String msg = requestBody == null ?
+        String.format("Unauthorized to perform %s %s", request.getMethod(), request.getURL()) :
+        String.format("Unauthorized to perform %s %s with body %s",
```

Comment:
Actually, this should not exist at all. Its being changed as part of https://github.com/caskdata/cdap/pull/5358. In this PR I was just trying to get contents of the body for debugging purposes, since the way it was before, it used just print the objects hashcode. This is supposed to get the logged in user, and not construct the message for the ``UnauthorizedException`` here. Instead it is supposed to just pass ``Principal``, ``EntityId`` and ``Set<Action>`` to ``UnauthorizedException``'s constructor. I'd prefer leaving it as is, since there is already another PR to fix this.